src/content/blog/whats-new-december-2023.mdx

```diff
@@ -0,0 +1,162 @@
+
+🧑‍🍳 Follow the recipe for [Quick-and-useful: A DIY, self-hosted Linktree and Linkinbio clone with Astro and TinaCMS](https://darko.io/posts/linktree-and-linkinbio-clone-with-astro-and-tinacms/) by @\_seiryuu\_
+
+⚠️ Get some tips, because maybe [You are using Astro wrong](https://arshx.substack.com/p/you-are-using-astro-wrong-a-survival) by @arshxcx
+
+🚀 Welcoming a new astronaut to the spaceship! A great article to send to your Astro-curious friends: [Launching a blog with Astro](https://goulven-clech.dev/2023/launching-blog-astro) by @goulvenclech
+
+📧 Learn how to use [using RESEND (email API) with Astro](https://www.youtube.com/watch?v=HyDwVN1AFwY&t=2s) in a new video by @chrispennington
+
+🎉 Read all about [A year working with HTML Web Components](https://hawkticehurst.com/writing/a-year-working-with-html-web-components/) by @hawkticehurst
+
+👀 Astro's 6-hour Launch Week Community Call Deluxe is on [YouTube](https://www.youtube.com/watch?v=VqfzAkI4AkY) hosted by @sarah11918 
+
+🛠️ Want to move [from Jekyll to Astro](https://jackcarey.co.uk/posts/astro-rewrite/)? Read this post by @jack_jack_jack
+
+🤿 Take a [Deep dive on View Transitions blog post series](https://fryuni.dev/deep-dive/astro-view-transitions/) by @Fryuni
+
+🏎️ New to Astro? Check out this updated [Astro Crash Course](https://www.youtube.com/watch?v=XoIHKO6AkoM) by Brad Traversy
+
+🏁 Define the controlled rollout of your next feature by following [Adding feature flags in Astro with Xata (showcasing View Transitions Forms)](https://xata.io/blog/feature-flags-tutorial) by @rishirajjain
+
+### Themes
+
+New Astro themes released this month. Try them out!
+
+- [Docbox](https://docbox-astro.vercel.app/) by @tffarhad
+- [ltree theme for Astro](https://github.com/DBozhinovski/astro-theme-ltree)
+- [Brutal](https://brutal.elian.codes) has reached over 150 stars by @eliancodes
+
+### Tools & tips
+
+With the release of the Astro Dev Toolbar, we also have Dev Toolbar apps to showcase now!
+
+- [Astro Dev Toolbar app for cloudflare](https://github.com/alexanderniebuhr/astro-cf-dev-toolbar-app-demo)
+- [Astro Tunnel app for Dev Toolbar](https://www.npmjs.com/package/astro-tunnel)
+- [Lighthouse app for Dev Toolbar](https://github.com/buk0vec/astro-lighthouse/tree/main)
+- [Cloudflare Dev Toolbar App (demo/WIP)](https://github.com/alexanderniebuhr/astro-cf-dev-toolbar-app-demo) by @alex
+- [Astro Font Picker devtoolbar app](https://github.com/randombits-dev/astro-font-picker) by @randombits.dev
+- [Astro Tunnel devtoolbar app](https://www.npmjs.com/package/astro-tunnel) by @shf0811
+- [astro-font](https://github.com/rishi-raj-jain/astro-font) by @rishirajjain
+- [Clerk + Astro demo](https://github.com/vin-e/clerk-astro-demo) by @vinny_code
+- [astro-heroicons](https://github.com/SeanMcP/astro-heroicons) updated by @seanmcp
+
+### Websites
+
+Websites submitted to our Discord `#showcase` channel and featured on community calls this month.
+
```

Comment:
Not sure how widely we include stuff here, but I’d guess this is a pretty big one from this month? ([Launch tweet](https://x.com/_davideast/status/1730361928285909250?s=20) for reference)

```suggestion

- [idx.dev/blog](https://idx.dev/blog) by Google
```

(This one would need to be added in #938 instead potentially)

Reply:
Oh yeah! thanks. Will add manually!